packages/editor/src/plugins/rows/components/add-row-button-floating.tsx

```diff
@@ -1,44 +1,48 @@
-import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
-import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
-import { useEditorStrings } from '@serlo/frontend/src/contexts/logged-in-data-context'
+import IconAddRow from '@editor/editor-ui/assets/plugin-icons/icon-add-row.svg'
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
 import { cn } from '@serlo/frontend/src/helper/cn'
 
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
 interface AddRowButtonFloatingProps {
-  onClick: React.MouseEventHandler<HTMLButtonElement>
+  onClick: React.MouseEventHandler<HTMLElement>
   focused?: boolean
   hide?: boolean
 }
 
 export function AddRowButtonFloating({
-  focused,
   onClick,
   hide,
 }: AddRowButtonFloatingProps) {
   const rowsStrings = useEditorStrings().plugins.rows
 
+  if (hide) return null
   return (
-    <div className="absolute bottom-0 z-[1] h-auto w-full translate-y-full">
-      {hide ? null : (
-        <div className="group flex justify-center">
-          <button
-            className={cn(
-              `
-              add-trigger z-20 h-7 w-7
-              rounded-full pt-1 text-center
-              text-gray-600 transition-all hover:cursor-pointer
-              hover:text-editor-primary hover:opacity-100
-              focus-visible:text-editor-primary group-hover:opacity-60
-              `,
-              focused ? 'opacity-60' : 'opacity-0'
-            )}
-            title={rowsStrings.addAnElement}
-            onClick={onClick}
-            data-qa="add-new-plugin-row-button"
-          >
-            <FaIcon icon={faCirclePlus} className="text-xl" />
-          </button>
+    <>
+      <div
+        className={cn(
+          'group ',
+          'opacity-0 transition-opacity duration-300',
+          'relative mb-4 flex items-center px-8 pb-2',
+          'text-editor-primary-200 hover:opacity-100',
+          'focus-visible:text-editor-primary-200 group-hover:opacity-100',
+          'cursor-pointer'
+        )}
+        onClick={onClick}
+      >
+        {/* Divider line */}
+        <div className="flex-grow border-t-2 border-gray-300"></div>
+        <div className={cn('group/btn serlo-tooltip-trigger relative px-6')}>
+          {/* Add button */}
+          <IconAddRow className="" />
+          <EditorTooltip
+            text={rowsStrings.addAnElement}
+            className="align-center"
+          />
         </div>
-      )}
-    </div>
+        {/* Divider line */}
+        <div className="flex-grow border-t-2 border-gray-300"></div>
```

Comment:
```suggestion
        <div className="flex-grow border-t-2 border-gray-300"/>
```
(preference, same below)